packages/forma-36-react-components/src/components/DateTime/DateTime.test.tsx

```diff
@@ -5,6 +5,22 @@ import { DateTime } from './DateTime';
 
 const exampleDate = new Date('2020-04-09T16:17:18.912Z');
 
+const monkeyPatch = (errorMessage: string) => {
```

Comment:
Okay then my last concern is that this should have a clearer name like `silenceErrorsWithMessage` or something.

Reply:
I like this wording people somehow came up with but I totally agree :)